.github/workflows/test.yml

```diff
@@ -51,10 +51,10 @@ jobs:
             pins: "h5py==3.7.* sqlalchemy==1.4.40 shapely==2.0.* pyproj==3.4.*"
             extras: "[test,gridadmin]"
           # current
-          - python: '3.10'  # h5py has no wheels for 3.11 (yet) https://github.com/h5py/h5py/issues/2146
+          - python: '3.11'
             os: ubuntu-latest
             numpy: numpy
-            pins: "git+https://github.com/nens/threedi-modelchecker@master#egg=threedi-modelchecker"
+            pins: "h5py>=3.8.* sqlalchemy>=2.0.* shapely==2.0.* pyproj==3.4.*"
```

Comment:
Please don't add version pins here; I want to test with the latest ones in the 'current'